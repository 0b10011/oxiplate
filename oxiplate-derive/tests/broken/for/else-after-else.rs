use oxiplate_derive::Oxiplate;

#[derive(Oxiplate)]
#[oxiplate_inline("
{% for number in numbers %}
    {{ number }}
{% else %}
    foo
{% else %}
    hello
{% else %}
    world
{% endfor %}
{% else %}")]
struct Data {
    foo: bool,
}

fn main() {
    print!("{}", Data { foo: true });
}
