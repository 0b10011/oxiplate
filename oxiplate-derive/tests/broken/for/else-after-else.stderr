error: proc-macro derive panicked
 --> tests/broken/for/else-after-else.rs:3:10
  |
3 | #[derive(Oxiplate)]
  |          ^^^^^^^^
  |
  = help: message: not yet implemented

error[E0277]: `Data` doesn't implement `std::fmt::Display`
  --> tests/broken/for/else-after-else.rs:20:18
   |
20 |     print!("{}", Data { foo: true });
   |             --   ^^^^^^^^^^^^^^^^^^ `Data` cannot be formatted with the default formatter
   |             |
   |             required by this formatting parameter
   |
   = help: the trait `std::fmt::Display` is not implemented for `Data`
   = note: in format strings you may be able to use `{:?}` (or {:#?} for pretty-print) instead
   = note: this error originates in the macro `$crate::format_args` which comes from the expansion of the macro `print` (in Nightly builds, run with -Z macro-backtrace for more info)
